packages/variable-editor/src/components/variables/master/VariablesMasterContent.tsx

```diff
@@ -71,6 +83,25 @@ export const VariablesMasterContent = () => {
     }
   });
 
+  const rows = table.getRowModel().rows;
+  const tableContainer = useRef<HTMLDivElement>(null);
+  const measureElement = (element: HTMLTableRowElement) => {
+    let height = ROW_HEIGHT;
+    let nextElement = element.nextElementSibling;
+    while (nextElement?.classList.contains('ui-message-row')) {
+      height += ROW_HEIGHT;
+      nextElement = nextElement.nextElementSibling;
+    }
+    return height;
+  };
+  const virtualizer = useVirtualizer<HTMLDivElement, HTMLTableRowElement>({
+    count: rows.length,
+    estimateSize: () => ROW_HEIGHT,
+    measureElement,
+    getScrollElement: () => tableContainer.current,
+    overscan: 5
```

Comment:
IMO we could also increase this value, so scrolling looks better, but didn't test it myself.